source/client/style/component/_tag.scss

```diff
@@ -5,13 +5,26 @@
   margin-bottom: 10px;
   margin-right: 10px;
   border-radius: 4px;
+  border-style: solid;
+  border-color: #DDDDDD;
+  border-width: 2px;
```

Comment:
1. You can use shorthand `border` property to set border style
    ```css
    border: 2px solid #DDDDDD;
    ```

2. Cannot add border here, at least not for every tag box. Because this will add border even to solid color tag box ![image](https://cloud.githubusercontent.com/assets/2153667/13021459/5394f7a4-d1a9-11e5-9aee-2084fc992162.png)